kalite/coachreports/api_views.py

```diff
@@ -13,6 +13,22 @@
 from kalite.shared.decorators.auth import require_admin
 from kalite.topic_tools.content_models import get_topic_contents, get_topic_nodes, get_leafed_topics, get_content_parents
 
+
+def unique_by_id_and_kind_sort(seq):
+
+    seq.sort(key=lambda x: x.get("sort_order", 0))
+    seen = {}
+    result = []
+    for item in seq:
+        marker = item.get("id") + item.get("kind")
```

Comment:
I can't imagine a circumstance where they could be equal, unless an item had a blank "kind" field.